core/src/main/java/org/apache/accumulo/core/classloader/ClassLoaderUtil.java

```diff
@@ -93,9 +93,8 @@ public static <U> Class<? extends U> loadClass(String className, Class<U> extens
   /**
    * Retrieve the classloader context from a table's configuration.
    */
-  @SuppressWarnings("removal")
   public static String tableContext(AccumuloConfiguration conf) {
-    return conf.get(conf.resolve(Property.TABLE_CLASSLOADER_CONTEXT, Property.TABLE_CLASSPATH));
+    return conf.get(conf.resolve(Property.TABLE_CLASSLOADER_CONTEXT));
```

Comment:
You should simplify these calls to `resolve`, if there's only one property. The `resolve` method tries to select the property to use based on what the user has set in the config. However, if there's only one, then there's nothing to `resolve` and you can just call `get` on the property directly.

```suggestion
    return conf.get(Property.TABLE_CLASSLOADER_CONTEXT);
```